Meshery-Adapters/linkerd-meshery-adapter/step3.md

```diff
@@ -46,12 +46,13 @@ Sample Output:
 |   web    |  1/1  |      1     |     1     | 4m38s |
 
 ### Let's inspect our pod:
+If you cannot see the *Ready* column updated, this is because the deployment can take up upto a minute. When all the deployment are in ready state, you can use `CTRL+ C` to switch back to the terminal.
```

Comment:
```suggestion
If you cannot see the *Ready* column updated, this is because the deployment can take up upto a minute. When all the deployment are in ready state, you can use `CTRL + C` or `CMD + C` to switch back to the terminal.
```

Reply:
Oops, I think I messed up by committing from the web itself 😞. How can I fix this?